docs/whatsnew/3.1.rst

```diff
@@ -34,5 +34,27 @@ We have bumped the minimum version of several packages we depend on; these are t
 
 .. _whatsnew-3.1-contributors:
 
+Changes to map date/time handling
+=================================
+
+New date properties
+-------------------
+The properties `~sunpy.map.GenericMap.date_beg`,
+`~sunpy.map.GenericMap.date_end`, and `~sunpy.map.GenericMap.date_avg` have
+been added to be drawn from the relevant FITS metadata, if present in the map
+header.
+
+Changes to `~sunpy.map.GenericMap.date`
+---------------------------------------
+`sunpy.map.GenericMap.date` now looks for more metadata than just DATE-OBS.
```

Comment:
Should we reference the FITS-4 standard in here somewhere to explain why we are making this change?

Reply:
I've updated the changelog and what's new with a bit of motivation.